include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -140,7 +141,26 @@ class BrushedMotorInterruptHandler {
         if (has_messages()) {
             update_and_start_move();
         } else if (motor_state == ControlState::POSITION_CONTROLLING) {
-            controlled_move_to(hold_encoder_position);
+            int32_t move_delta =
+                hardware.get_encoder_pulses() - hold_encoder_position;
+            controlled_move_to(move_delta);
+            // we use double the acceptable position here just to allow the pid
+            // loop the oppertunity to maintain small movements that occure from
+            // motion and vibration
+            if (move_delta > 2 * acceptable_position_error) {
```

Comment:
maybe make this a named constant? it's easy to miss the digit in all the variable names.